packages/element-app/src/components/WebKeystore/KeystoreCard.js

```diff
@@ -0,0 +1,372 @@
+              </IconButton>
+            )}
+
+            {status === 'unlocked' && (
+              <IconButton
+                className={clsx(classes.expand, {
+                  [classes.expandOpen]: expanded,
+                })}
+                onClick={handleExpandClick}
+                aria-expanded={expanded}
+                aria-label="show more"
+              >
+                <ExpandMoreIcon />
+              </IconButton>
+            )}
+          </CardActions>
+
+          <Collapse in={expanded} timeout="auto" unmountOnExit>
+            <CardContent>
+              {status === 'unlocked' &&
+                Object.keys(keystore.data.keys).map(kid => {
+                  // eslint-disable-next-line security/detect-object-injection
+                  const key = keystore.data.keys[kid];
+
+                  return (
+                    <ExpansionPanelList
+                      key={kid}
+                      panels={[
+                        {
+                          title: `${key.tags[0]} ${kid.substring(0, 8)}...`,
+                          children: (
+                            <div style={{ width: '100%' }}>
+                              <Typography variant="body2">
+                                {key.notes}
+                              </Typography>
+                              <br />
+
+                              <div>
+                                {key.tags.map(t => (
+                                  <CopyToClipboard
+                                    key={t}
+                                    text={t}
+                                    onCopy={() => {
+                                      // eslint-disable-next-line no-alert
+                                      alert(`Copied ${t} to clipboard`);
+                                    }}
+                                  >
+                                    <Chip
+                                      key={t}
+                                      label={t}
+                                      variant="outlined"
+                                      style={{ margin: '4px' }}
+                                    />
+                                  </CopyToClipboard>
+                                ))}
+                              </div>
+                              <br />
+                              <CopyToClipboard
+                                text={key.publicKey}
+                                onCopy={() => {
+                                  // eslint-disable-next-line no-alert
+                                  alert('Copied public key to clipboard');
+                                }}
+                              >
+                                <Button
+                                  style={{ marginTop: '16px' }}
+                                  fullWidth
+                                  variant="contained"
+                                >
+                                  Copy Public Key
```

Comment:
Not working for bip39 mnemonic

Reply:
I made it not possible to copy it, you can see it in the editor though.